### ISSUE TITLE
'SyntaxError: invalid syntax' error with python3.7

### ISSUE THREAD

**Issue by JV-X** [hack4impact/flask-base#184]
i clone code to my mac and try start by README.md, but when i run command 
`python3 manage.py recreate_db`
this issues appear:

```
Traceback (most recent call last):
  File "manage.py", line 8, in <module>
    from rq import Connection, Queue, Worker
  File "/usr/local/lib/python3.7/site-packages/rq/__init__.py", line 9, in <module>
    from .queue import get_failed_queue, Queue
  File "/usr/local/lib/python3.7/site-packages/rq/queue.py", line 58
    async=True, job_class=None):
        ^
SyntaxError: invalid syntax
```

OS: MacOS Mojave 10.14.2
Python Version: 3.7

**Comment by scollar71** [comment]
rq 0.7.1 not compatible with py 3.7 currently 1.0 update requirements.txt to reflect

**Comment by JV-X** [comment]
thank you for your effort

**Comment by songpon** [comment]
Hi I faced the same problem with MAC python 3.7.3
please use this
pip install https://github.com/rq/rq/archive/v1.1.0.tar.gz

**Comment by nk9** [comment]
Just to be clear, the solution to this error is:

```
pip install --upgrade rq
pip freeze --local > requirements.txt
```

**Comment by abhisuri97** [comment]
Should be fixed due to PR #193. closing.
